src/pkg/packager/compose.go

```diff
@@ -139,11 +139,14 @@ func (p *Packager) fixComposedFilepaths(parent, child types.ZarfComponent) types
 		child.Files[fileIdx].Source = p.getComposedFilePath(file.Source, parent.Import.Path)
 	}
 
-	// Prefix non-url composed component chart values files.
+	// Prefix non-url composed component chart values files and localPath.
 	for chartIdx, chart := range child.Charts {
 		for valuesIdx, valuesFile := range chart.ValuesFiles {
 			child.Charts[chartIdx].ValuesFiles[valuesIdx] = p.getComposedFilePath(valuesFile, parent.Import.Path)
 		}
+		if child.Charts[chartIdx].LocalPath != "" {
```

Comment:
Just for my own education, is this the proper way to check this variable when dealing with an `omitempty` field?
```go
LocalPath   string   `json:"localPath,omitempty" jsonschema:"oneof_required=localPath,description=The path to the chart folder"`
```

Reply:
Yep! That looks correct assuming you are checking that LocalPath has not been omitted (i.e. it contains a value).  By default (when not using pointers) Go will default to the zero value for that type https://go.dev/tour/basics/12